### DISCUSSION TITLE
`q:`

### DISCUSSION THREAD

**Discussion by semi710** [folke/noice.nvim#534] [Q&A]
how to get noice prompt for `q:` cmd prompt where it's easier to edit the text by going in normal mode

or if switching to normal mode is possible in cmd prompt?

**Comment by folke** [comment]
That's not possible

**Reply by semi710** [reply]
maybe this is out of context.
but some how i found that wokring in here https://github.com/nyoom-engineering/nyoom.nvim

don't know how but theirs command popup supports mode switching.
